docs/_documentations/workingwithtemplates.md

```diff
@@ -16,6 +16,8 @@ Codewind gives you the flexibility to code and develop in languages of your choi
 
 When you create a new project, you can choose from the default set of templates available in Codewind, or you can choose a template source of your own. You can add more development languages and sample projects by adding new templates. 
 
+Codewind provides pre-configured, Docker-containerized project templates covering several languages including [Node.js](https://nodejs.dev/), [Java](https://www.java.com/), [Python](https://www.python.org/) and [Swift](https://swift.org/), and several frameworks such as [Express](https://expressjs.com/), [Spring Boot](https://spring.io/projects/spring-boot) and [Open Liberty](https://openliberty.io/). [Quickly create and deploy microservices](https://www.youtube.com/watch?v=zKMggp10gq4&t=12s) in languages and frameworks that you're familiar with. You can easily modify these pre-configured projects to develop your customised microservices.
```

Comment:
- Terminology question: Last year, I think I heard that we should use "container" instead of specifying a Docker container to make our language more inclusive. If this is still the case, we should say, "containerized project templates" instead of specifying "Docker-containerized."
- Comma after "several languages"
- Comma after `[Python](https://www.python.org/)`. This is an Oxford comma.
- Comma after  "and several frameworks"
- Comma after `[Spring Boot](https://spring.io/projects/spring-boot)`. This is an Oxford comma.
- "Preconfigured" should not have a hyphen.
- "Customized" should be spelled with a "z" for the American spelling.

Reply:
- Changed to containerized
- Added comma.
- Added Oxford comma.
- Added comma after "and several frameworks".
- Added `Spring Boot` Oxford comma.
- Removed hyphen
- Changed spelling